crates/adapters/dydx/src/http/parse.rs

```diff
@@ -1344,7 +1344,7 @@ mod reconciliation_tests {
             total_filled: dec!(1.0),
             price: dec!(50000.0),
             status: DydxOrderStatus::PartiallyFilled,
-            order_type: "Limit".to_string(), // EnumString uses PascalCase
+            order_type: "LIMIT".to_string(),
```

Comment:
We could use the `DydxOrderType` enum here since it's serializable (for a follow-up).